tutor/src/screens/teacher-dashboard/month.jsx

```diff
@@ -243,26 +244,26 @@ class CourseMonth extends React.Component {
       this.context.router.history.push(url)
     } else { // is a task plan to clone
       this.cloningPlan = extend({}, item, {
```

Comment:
Modifying the `cloningPlan` observable is causing this:

![screen shot 2018-04-12 at 12 44 46 pm](https://user-images.githubusercontent.com/2483873/38694500-6f785da2-3e4f-11e8-883a-c075610b15df.png)

Reply:
yeah, I'm still not entirely sure what triggered that.  I "solved" it by returning a new object from the drop, instead of extending the existing observable object